src/getHumanAge.test.js

```diff
@@ -7,4 +7,52 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should have right length', () => {
+    const result = getHumanAge(2, 3);
+
+    expect(result).toHaveLength(2);
```

Comment:
The test checks if the result has a length of 2, which assumes that `getHumanAge` returns an array. Ensure that the function is expected to return an array of length 2 as per the task description.